pkg/virt-launcher/virtwrap/converter/network.go

```diff
@@ -45,7 +45,10 @@ func createDomainInterfaces(vmi *v1.VirtualMachineInstance, domain *api.Domain,
 
 	networks := indexNetworksByName(vmi.Spec.Networks)
 
-	for i, iface := range vmi.Spec.Domain.Devices.Interfaces {
+	if len(ifacesToHotplug) == 0 {
+		ifacesToHotplug = vmi.Spec.Domain.Devices.Interfaces
```

Comment:
Since it is reused for both hotplug and normal stratup, I would call it `interfaceToPlug` (same for all occurrences, also phase1).

Reply:
I named this plural for one simple reason (not saying it's a good one ...): I find it very puzzling when iterating  variables if their name is singular - i.e. `for i, iface := range ifaceToHotplug {...}` . 

I guess both are awkward ...

Reply:
My suggestion was singular by mistake. I just wanted you to remove the `hot`.